plugins/image-prioritizer/class-image-prioritizer-background-image-styled-tag-visitor.php

```diff
@@ -35,11 +35,11 @@ public function __invoke( OD_HTML_Tag_Walker $walker ): bool {
 		 * parser of the `url()` functions from the property value.
 		 */
 		$background_image_url = null;
-		$style                = $walker->get_attribute( 'style' );
+		$style                = $processor->get_attribute( 'style' );
 		if (
 			is_string( $style )
 			&&
-			1 === preg_match( '/background(?:-image)?\s*:[^;]*?url\(\s*[\'"]?\s*(?<background_image>.+?)\s*[\'"]?\s*\)/', $style, $matches ) // Not compatible with strict rules. See <https://github.com/phpstan/phpstan/issues/11262>.
```

Comment:
This comment was supposed to have been removed in https://github.com/WordPress/performance/pull/1325.